packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/disable-security-image/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Disable the security image and additional self-service recovery options
+---
+
+With Okta, new users are prompted to choose a security image and optionally an additional security question prompt when they first sign in. To provide your users with a seamless and smooth sign-in experience, you may need to disable these options.
+
+1. In the Admin Console, go to **Settings** and then **Customization**.
+2. In the **Optional User Account Fields** section, click **Edit**.
+3. Select **Disabled** from the **Security image** drop-down box, and then click **Save**.
+
+> **Note:** If your org doesn't allow disabling additional self-service recovery options, you can skip the following steps. Disabling additional self-service recovery options is an Early Access feature. To enable it, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
+
+1. Go to **Security** and then **Authentication**
```

Comment:
Classic UI only

Reply:
@briandemers-okta - They can't use the Developer Console for these steps. They have to use the Admin Console (Classic UI). I will put a note at the very top about performing these steps in the Admin Console so that users aren't confused.